deals/deals_test.go

```diff
@@ -28,27 +35,24 @@ func TestStore(t *testing.T) {
 	numMiners := []int{1}
 	for _, nm := range numMiners {
 		t.Run(fmt.Sprintf("CantMiners%d", nm), func(t *testing.T) {
-			dnet, _, _, close := tests.CreateLocalDevnet(t, nm)
-			defer close()
-			m, err := New(dnet.Client, WithImportPath(filepath.Join(os.TempDir(), "imports")))
+			client, _, _ := tests.CreateLocalDevnet(t, nm)
+			m, err := New(client, WithImportPath(filepath.Join(tmpDir, "imports")))
 			checkErr(t, err)
-			_, err = storeMultiMiner(m, dnet, nm, randomBytes(1000))
+			_, err = storeMultiMiner(m, client, nm, randomBytes(1600))
 			checkErr(t, err)
 		})
 	}
 }
-
 func TestRetrieve(t *testing.T) {
 	numMiners := []int{1} // go-fil-markets: doesn't support remembering more than 1 miner
-	data := randomBytes(1000)
+	data := randomBytes(1600)
```

Comment:
Magic number changes for `testnet/3` sectorbuilder-mock. 
Nothing too much important since this data is random... the size shouldn't be meaningful.